lib/browser/client-scripts/calibrate.js

```diff
@@ -48,10 +48,13 @@
     }
 
     function getBrowserFeatures() {
+        var iOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
```

Comment:
you don't need `MSStream` here - for IE userAgent is `"Mozilla/5.0 (Windows NT 6.3; Trident/7.0; .NET4.0E; .NET4.0C; rv:11.0) like Gecko"`

Reply:
I will remove it. Just copied it from some place.